ui/src/components/StickerSidebar/StickerSidebar.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import barcode from './barcode.png';
+import parentalAdvisory from './parental-advisory.png';
```

Comment:
Another way to support this is to store these somewhere else, i.e. database or use a cloud service provider, and get the source to this through an http request. Doing it this way would mean that to add more stickers, you would have to add them to the repo, which isn't scalable.